ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/headervalidationrules/ProofOfWorkValidationRule.java

```diff
@@ -95,6 +106,10 @@ Hash hashHeader(final BlockHeader header) {
     out.writeLongScalar(header.getGasUsed());
     out.writeLongScalar(header.getTimestamp());
     out.writeBytes(header.getExtraData());
+    if (includeBaseFee && header.getBaseFee().isPresent()) {
```

Comment:
If we are to include the basefee shouldn't we require the header have the base fee?  Or does the spec allow it to be absent?  i.e. if we include base fee should an absent base fee be marked as invalid?

Reply:
Yes you are right. The spec does not allow it to be absent. So i will do the change to make it required if the boolean is set to true.